src/coreclr/tools/Common/TypeSystem/Ecma/EcmaType.cs

```diff
@@ -429,6 +429,24 @@ public override IEnumerable<FieldDesc> GetFields()
             }
         }
 
+        public override TypeDesc UnderlyingType
```

Comment:
Maybe delete the TODO here:

https://github.com/dotnet/runtime/blob/4ff5a3a85a9b0de7ab9e9267959e668142815f4f/src/coreclr/tools/Common/TypeSystem/Common/TypeDesc.cs#L475

If we're doing overrides, we're unlikely to benefit from a cache there.

Reply:
Good point. I've removed the comment.